src/Ubiquity/controllers/crud/CRUDControllerUtilitiesTrait.php

```diff
@@ -187,7 +187,7 @@ protected function getModelViewer(): ModelViewer {
 		return new ModelViewer ( $this );
 	}
 
-	private function _getModelViewer(): ModelViewer {
+	protected function _getModelViewer(): ModelViewer {
```

Comment:
It is also necessary to change the visibility in **MessagesTrait**:
https://github.com/phpMv/ubiquity/blob/70e4321964c7ccc6b9458c4c8509ee968a03b704/src/Ubiquity/controllers/semantic/MessagesTrait.php#L25